tensorflow_datasets/core/dataset_builder.py

```diff
@@ -772,11 +828,12 @@ def _make_download_manager(self, download_dir, download_config):
     )
 
   @property
-  def builder_config(self):
+  def builder_config(self) -> BuilderConfig:
     """`tfds.core.BuilderConfig` for this builder."""
     return self._builder_config
 
-  def _create_builder_config(self, builder_config):
+  def _create_builder_config(self,
+                             builder_config: BuilderConfig) -> Optional[BuilderConfig]:  # noqa
```

Comment:
What is the `# noqa` comment about ?

Reply:
Sorry, When I first made the commit I was checking not `pylint`, but I was checking `pycodestyle`. That's why. In `pycodestyle`, to ignore line-too-long, `#noqa` is used.